examples/tv-casting-app/tv-casting-common/src/CastingServer.cpp

```diff
@@ -168,6 +168,7 @@ CastingServer::GetDiscoveredCommissioner(int index, chip::Optional<TargetVideoPl
 void CastingServer::ReadServerClustersForNode(NodeId nodeId)
 {
     ChipLogProgress(NotSpecified, "ReadServerClustersForNode nodeId=0x" ChipLogFormatX64, ChipLogValueX64(nodeId));
+    Init();
```

Comment:
Init seems to be `CHIP_ERROR CastingServer::Init`. Could we add error checking here and in the other newly added init call sites? Even if it is just a `log on error`

Reply:
added logging of any error encountered